test/pdo_sqlsrv/pdo_azure_ad_authentication.phpt

```diff
@@ -20,15 +20,15 @@ catch( PDOException $e )
     echo "\n";
 }
 
-$stmt = $conn->query( "SELECT name FROM master.dbo.sysdatabases" );
+$stmt = $conn->query( "SELECT count(*) FROM cd_info" );
```

Comment:
Why change from fetching string to integer? It may change what the test is originally testing (although I'm sure our driver has no problem fetching strings)